integration-cli/docker_cli_logs_test.go

```diff
@@ -276,6 +277,76 @@ func (s *DockerSuite) TestLogsFollowStopped(c *check.C) {
 	deleteContainer(cleanedContainerID)
 }
 
+func (s *DockerSuite) TestLogsSince(c *check.C) {
+	name := "testlogssince"
+	runCmd := exec.Command(dockerBinary, "run", fmt.Sprintf("--name=%s", name), "busybox", "/bin/sh", "-c", `date +%s; for i in $(seq 1 5); do sleep 1; echo log$i; done`)
```

Comment:
For next time, you can make it: `"--name", name,` instead of using the Sprintf stuff.